include/cppkafka/kafka_handle_base.h

```diff
@@ -340,6 +340,30 @@ class CPPKAFKA_API KafkaHandleBase {
      */
     int get_out_queue_length() const;
 
+#if RD_KAFKA_VERSION >= RD_KAFKA_DESTROY_FLAGS_SUPPORT_VERSION
+    /**
+     * \brief Sets flags for rd_kafka_destroy_flags()
+     *
+     * 0 (default) - calls consumer_close() during handle destruction
```

Comment:
I would only keep the `brief` message. The rest is too specific for one particular flag and there are other flags also.